apps/vanilla/src/pages/exercise2/scripts/DiceGenerator.ts

```diff
@@ -0,0 +1,25 @@
+import { Publisher } from './Publisher';
+import { ICurrentPlayerMoveData, ISubscriber, IMoveData } from './types';
+
+/** Random dice generator. */
+export class DiceGenerator extends Publisher<IMoveData> implements ISubscriber<ICurrentPlayerMoveData> {
+
+	/** Function update data.
+	 * @param data - Walking player data.
+	 */
```

Comment:
We follow this format for jsdoc:

```suggestion
	/** 
   * Function update data.
	 * @param data Walking player data.
	 */
```